File: content/en/docs/Tutorials/Sensors/Emitter and Receiver/scoring.c
char message[9]; // Here we use a 9 byte array, since sizeof(int + int + char) = 9
sprintf(message, "%d%d%c", 0, 0, 'H'); // Format the array with the data X-Pos = 0 cm, Z-Pos = 0 cm, victim type = harmed 
    
emitter->send(message, sizeof(message)); // Send out the message array. Note that the 2nd parameter must be the size of the message

File: content/en/docs/Tutorials/Sensors/Emitter and Receiver/scoring.py
from controller import Robot, Receiver, Emitter
import struct # Use the struct module in order to pack the message sent to the Main Supervisor

...
    
victimType = bytes('H', "utf-8") # The victim type being sent is the letter 'H' for harmed victim
message = struct.pack("i i c", 0, 0, victimType) # Pack the message. The estimated position of the victim is 0 cm, 0 cm.
    
emitter.send(message) # Send out the message